include/flox/resolver/manifest.hh

```diff
@@ -119,6 +121,178 @@ from_json( const nlohmann::json & jfrom, ManifestRaw & manifest );
 // void to_json( nlohmann::json & jto, const ManifestRaw & manifest );
 
 
+/* -------------------------------------------------------------------------- */
+
+class UnlockedManifest
```

Comment:
Can you add a docstring somewhere that explains what the difference is between a Manifest, a ManifestRaw, and an UnlockedManifest?

Reply:
https://imgur.com/gallery/NkAY5KS

Reply:
Yeah tho I will during regular business hours

Reply:
You're just witnessing iteration in action. I'll end up killing 2/3 of these structures later

Reply:
Alright, I'll approve for now, just remember there's other people that will have to decipher once you're on PTO so leave us some breadcrumbs

Reply:
Oh I know, and I'll do everything my faculties will allow to help them 🖤